src/mantine-dates/src/components/Month/get-day-tabindex/get-day-tabindex.ts

```diff
@@ -0,0 +1,30 @@
+import dayjs from 'dayjs';
+import { isSameMonth } from '../is-same-month/is-same-month';
+
+interface Props {
+  date: Date;
+  isSelected: boolean;
+  month: Date;
+  hasSelectedDayInMonth: boolean;
+}
+
+export function getDayTabIndex({ date, isSelected, hasSelectedDayInMonth, month }: Props) {
+  if (isSelected) {
+    return 0;
+  }
+
+  const hasTodayInMonth = isSameMonth(new Date(), month);
+
+  const isToday = dayjs(date).isSame(new Date(), 'day');
+  if (!hasSelectedDayInMonth && isToday) {
+    return 0;
+  }
+
+  const isOutside = !isSameMonth(date, month);
+  const isFirstDayInMonth = dayjs(date).format('DD') === '01' && !isOutside;
+  if (!hasSelectedDayInMonth && !hasTodayInMonth && isFirstDayInMonth) {
```

Comment:
What if the first day of moth is disabled? For example, it is before min date, after max date, disabled with `excludeDate` or getDayProps makes it disabled.

Reply:
With the new commit I account for these things.